hw/ip/spi_device/dv/spi_device_sim_cfg.hjson

```diff
@@ -41,6 +41,9 @@
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
 
+  // Overwrite timescale to coarse
+  timescale: "1ns/1ns"
```

Comment:
we use this way to override a field. Here is an [example](https://github.com/lowRISC/opentitan/blob/d9a7c02c1880cc5c8c58f0d51c65003ee3152551/hw/top_earlgrey/dv/chip_sim_cfg.hjson#L52)
```
  overrides: [
    {
      name: timescale
      value: "1ns/1ns"
    }
 ]
```

Reply:
can you explain where you use 1ns precision?

Reply:
Revised to use `overrides`. `1ns` is an example to check if it works or not.